charts/portworx/README.md

```diff
@@ -35,16 +35,18 @@ The following tables lists the configurable parameters of the Portworx chart and
 |             Parameter       |            Description             |                    Default                |
 |-----------------------------|------------------------------------|-------------------------------------------|
 | `deploymentType`            | The deployment type. Can be either docker/oci   | `oci`                 |
-| `imageVersion`              | The image tag to pull              | `2.0.3.4`                                  |
+| `imageVersion`              | The image tag to pull              | `2.1.2`                                  |
 | `openshiftInstall`               | Installing on Openshift? | `false`                               |
 | `pksInstall`               | Installing on Pivotal Container service? | `false`                               |
-| `AKSorEKSInstall`               | Installing on AKS(Azure Kubernetes service) or EKS (Amazon Elastic Container service) | `false`                               |
+| `EKSInstall`               | Installing EKS (Amazon Elastic Container service) | `false`                               |
+| `AKSInstall`               | Installing on AKS(Azure Kubernetes service) | `false`                               |
 | `etcdEndPoint`          | (REQUIRED) ETCD endpoint for PX to function properly in the form "etcd:http://<your-etcd-endpoint>". Multiple Urls should be semi-colon seperated example: etcd:http://<your-etcd-endpoint1>;etcd:http://<your-etcd-endpoint2>  | `etcd:http://<your-etcd-endpoint>`                    |
 | `clusterName`           | Portworx Cluster Name  | `mycluster`                                     |
 | `usefileSystemDrive`      | Should Portworx use an unmounted drive even with a filesystem ? | `false`                |
 | `usedrivesAndPartitions`  | Should Portworx use the drives as well as partitions on the disk ? | `false`             |
 | `secretType`      | Secrets store to be used can be AWS KMS/KVDB/Vault/K8s/IBM Key Protect          | `none`                                    |
 | `drives` | Semi-colon seperated list of drives to be used for storage (example: "/dev/sda;/dev/sdb")           | `none`                                   |
+| `defaultDrives` | Defaults for AKS installation if drives are not provided           | `type=Premium_LRS,size=150`                                   |
```

Comment:
`defaultDrives` doesn't need to be user facing. This will confuse non-AKS users.

Just let the AKSInstall drive the logic and internally we pick the defaults.

Reply:
makes sense